.github/actions/generic-ci/action.yml

```diff
@@ -150,6 +163,7 @@ runs:
         path: ${{github.workspace}}\build\bin_Release
 
     - name: Test
+      if: inputs.cpu == 'x86_64'
```

Comment:
afaiu arm just cannot be tested. Wdyt of adding a new inputs var to control this behavior instead of checking the cpu arch ? It would be more clear in terms of intent.

Reply:
Hmm. That means we will have two inputs that will be enabled strictly at the same time. I'm not against it but it will add bloating code.  
What I can do is adding a new step which set an env variable like `ENABLE_TESTING` depending on criteria (will only depend on the cpu arch for now)

Reply:
that could work

Reply:
used ga outputs instead of envvar